src/client/utils/dom.js

```diff
@@ -2,10 +2,28 @@ import CONST from '../../const';
 import trim from '../../utils/string-trim';
 import nativeMethods from '../sandbox/native-methods';
 import urlUtils from '../utils/url';
-import { isMozilla } from './browser';
+import { isMozilla, isWebKit, isIE, isOpera } from './browser';
 
 var scrollbarSize = null;
 
+function getFocusableSelector () {
+    //NOTE: We don't take into account the case of embedded contentEditable elements and
+    // specify the contentEditable attribute for focusable elements
+    var selectorPostfix = 'input, select, textarea, button, [contenteditable="true"], [contenteditable=""], [tabIndex]';
+
+    if (isIE)
+        return 'a[href]:not([href = ""]), iframe, ' + selectorPostfix;
+
+    if (isOpera)
+        return selectorPostfix;
+
+    return 'a[href], iframe, ' + selectorPostfix;
+}
+
+function isHidden (element) {
```

Comment:
There is no global `element` var like we have with `window` and `document`, so it's just fine